zerver/views/home.py

```diff
@@ -228,6 +229,8 @@ def home_real(request):
         realm_restricted_to_domain = register_ret['realm_restricted_to_domain'],
         realm_default_language = register_ret['realm_default_language'],
         realm_waiting_period_threshold = register_ret['realm_waiting_period_threshold'],
+        realm_icon=realm_icon_url(user_profile.realm),
+        realm_icon_source=user_profile.realm.icon_source,
```

Comment:
Also, do we need `realm_icon_source` on the frontend?  I would think we'd only need the URL...